toolbox.go

```diff
@@ -0,0 +1,44 @@
+package trails
```

Comment:
I'm curious what the rationale was for this sort of thing needing to live in `trails` vs the parent app?

Reply:
The intention is to use it in `second-child`.